angular.json

```diff
@@ -28,16 +28,6 @@
             "tsConfig": "projects/imgix-angular/tsconfig.spec.json",
             "karmaConfig": "projects/imgix-angular/karma.conf.js"
           }
-        },
-        "lint": {
-          "builder": "@angular-devkit/build-angular:tslint",
-          "options": {
-            "tsConfig": [
-              "projects/imgix-angular/tsconfig.lib.json",
-              "projects/imgix-angular/tsconfig.spec.json"
-            ],
-            "exclude": ["**/node_modules/**"]
-          }
```

Comment:
I'm OOL, why do we need to remove the `lint` here?

Reply:
TSLint was deprecated in Angular 11 and removed in Angular 13. See https://github.com/angular/angular-cli/issues/20105